bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -17,6 +17,9 @@ metadata:
           },
           "spec": {
             "size": "starterset"
+          },
```

Comment:
May I ask that is this added by operator-sdk automatically?

Reply:
I added it, just tested and found it could be removed

Reply:
We probably need to remove it. It's a little bit strange to include a status when creating CR